drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -117,20 +149,24 @@
             </header>
             <section class="tab-content">
                 <form class="item-styles-form">
-                    <label class="h3" for="item-background-color">{% trans "Background color" %}</label>
-                    <input id="item-background-color"
-                           placeholder="e.g. blue or #0000ff"
-                           value="{{ self.item_background_color}}"
-                           aria-describedby="item-background-color-description">
-                    <div id="item-background-color-description" class="item-styles-form-help">
+                    <label class="h3">
+                        <span>{% trans self.fields.item_background_color.display_name %}</span>
+                        <input class="item-background-color"
+                               placeholder="e.g. blue or #0000ff"
```

Comment:
Needs i18n. `blue` is a CSS keyword so shouldn't be translated, but "e.g." is English-specific. So this should be marked for i18n as

```
{% blocktrans with example1="blue" example2="#0000ff" %}
e.g. {{example1}} or {{example2}}
{% endblocktrans %}
```